app/src/main/kotlin/com/hippo/ehviewer/spider/SpiderDen.kt

```diff
@@ -218,6 +223,22 @@ class SpiderDen(mGalleryInfo: GalleryInfo) {
             override fun close() {}
         }
     }
+
+    suspend fun exportAsCbz(file: UniFile) = resourceScope {
+        val pages = info.pages
+        val comicInfo = closeable { (downloadDir!! / COMIC_INFO_FILE).openFileDescriptor("r") }
+        val (fdBatch, names) = (0 until pages).parMap { idx ->
+            val ext = requireNotNull(getExtension(idx))
+            val f = autoCloseable { requireNotNull(getImageSource(idx)) }
```

Comment:
`getImageSource` will only read from cache since `mode` is read, use `findImageFile` instead.